engine/schema/templateConfig.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
is it good to put file in engine/schema ?

Reply:
placed it here, as it the files generated by this script is consumed by SystemVMTemplateRegistration to populate details such as checksum, download path, template name, etc during upgrade